packages/strapi-plugin-content-manager/admin/src/containers/CollectionTypeWrapper/index.js

```diff
@@ -93,7 +111,7 @@ const CollectionTypeWrapper = ({ allLayoutData, children, from, slug }) => {
 
         dispatch({
           type: 'GET_DATA_SUCCEEDED',
-          data: cleanReceivedData(data),
+          data: cleanReceivedData(cleanClonedData(data)),
```

Comment:
Why do we need to run those two functions? 
it seems that `cleanClonedData(data)` should be enough

Reply:
The cleanReceivedData sanitises all password fields (in case the API returns them) and also add the components a unique key to all repeatable components in order to make the DND works.